### DISCUSSION TITLE
作为一个新手,请教一下配置自定义响应的问题

### DISCUSSION THREAD

**Discussion by ipanocloud** [apache/apisix#7494] [Q&A]
### Description

请教一下,APISIX2.2 怎么配置出 暂无可用节点的这种自定义响应？

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

**Comment by tzssangglass** [comment]
I don't really understand your question
